src/NumberField/NumberField.test.tsx

```diff
@@ -48,7 +48,7 @@ describe("NumberField", () => {
     expect(inputBase.value).toBe("123");
   });
   test("Numberfield has a default value", () => {
-    const { container } = render(<NumberField defaultValue={22} />);
+    const { container } = render(<NumberField value={22} />);
```

Comment:
Doesn't this defeat the point of the test, which is to prove that we support defaultValue as a prop?